types/hapi/definitions/route/route-options-response.d.ts

```diff
@@ -0,0 +1,84 @@
+import {Lifecycle, Util} from "hapi";
+import {ValidationOptions} from "joi";
+
+/**
+ * For context [See docs](https://github.com/hapijs/hapi/blob/master/API.md#-routeoptionsresponseemptystatuscode)
+ */
+export enum EmptyStatusCode {
+    HTTP_200 = 200,
```

Comment:
This seems silly, if you want to enum status codes why not go with `Ok`, `NoContent` rather than just repeating the values.

Reply:
Agreed, I got rid of it.